tests/statements/ddl.rs

```diff
@@ -275,7 +284,12 @@ async fn test_create_table_move_and_drop(
 #[rstest]
 #[tokio::test]
 async fn test_create_table_drop_schema(
-    #[values(ObjectStoreType::InMemory, ObjectStoreType::Local, ObjectStoreType::S3)]
+    #[values(
+        ObjectStoreType::InMemory,
+        ObjectStoreType::Local,
+        ObjectStoreType::S3(None),
+        ObjectStoreType::S3(Some("/path/to/folder"))
```

Comment:
Does this work with or without the leading slash?

Reply:
Yup, works in both cases, the `test_table_location_s3` actually covers that.